src/main/java/com/mapzen/valhalla/Route.kt

```diff
@@ -10,22 +9,53 @@ import java.util.ArrayList
 import java.util.HashSet
 
 public open class Route {
-    public val SNAP_PROVIDER: String = "snap"
-    public val LOST_THRESHOLD: Int = 50
-    public val CLOCKWISE: Int = 90
-    public val COUNTERCLOCKWISE: Int = -90
-    public val CORRECTION_THRESHOLD: Int = 1000
-    public val REVERSE: Int = 180
+
+    private val TAG = Route::class.java.simpleName
+
+    companion object {
+        public const val KEY_TRIP = "trip"
+        public const val KEY_LEGS = "legs"
+        public const val KEY_SHAPE = "shape"
+        public const val KEY_MANEUVERS = "maneuvers"
+        public const val KEY_UNITS = "units"
+        public const val KEY_LENGTH = "length"
+        public const val KEY_STATUS = "status"
+        public const val KEY_TIME = "time"
+        public const val KEY_LOCATIONS = "locations"
+        public const val KEY_SUMMARY = "summary"
+        public const val SNAP_PROVIDER: String = "snap"
+        public const val CLOSE_TO_DESTINATION_THRESHOLD_METERS: Int = 20
+        public const val CLOSE_TO_NEXT_LEG_THRESHOLD_METERS: Int = 5
+        public const val LOST_THRESHOLD_METERS: Int = 50
+        public const val CORRECTION_THRESHOLD_METERS: Int = 1000
+        public const val CLOCKWISE_DEGREES: Double = 90.0
+        public const val COUNTERCLOCKWISE_DEGREES: Double = -90.0
+        public const val REVERSE_DEGREES: Int = 180
+        public const val LOCATION_FUZZY_EQUAL_THRESHOLD_RADIANS: Double = 0.00001
```

Comment:
Since this is being used to compare lat/lng values I believe the units should be degrees instead of radians here.

Also of note is that my browser spellcheck does not recognize the word "radian" (or its plural) by default.